qiskit/pulse/schedule.py

```diff
@@ -246,9 +246,78 @@ def filter(self, *filter_funcs: List[Callable],
                           return a bool
             channels: For example, [DriveChannel(0), AcquireChannel(0)]
             instruction_types: For example, [PulseInstruction, AcquireInstruction]
-            time_ranges: Time intervals to keep, e.g. [(0, 5), (6, 10)]
-            intervals: Time intervals to keep, e.g. [Interval(0, 5), Interval(6, 10)]
+            time_ranges: For example, [(0, 5), (6, 10)]
+            intervals: For example, [Interval(0, 5), Interval(6, 10)]
+        """
+        composed_filter = self._construct_filter(*filter_funcs,
+                                                 channels=channels,
+                                                 instruction_types=instruction_types,
+                                                 time_ranges=time_ranges,
+                                                 intervals=intervals)
+        return self._apply_filter(composed_filter,
+                                  new_sched_name="{name}-filtered".format(name=self.name))
+
+    def exclude(self, *filter_funcs: List[Callable],
+                channels: Optional[Iterable[Channel]] = None,
+                instruction_types: Optional[Iterable[Type['Instruction']]] = None,
+                time_ranges: Optional[Iterable[Tuple[int, int]]] = None,
+                intervals: Optional[Iterable[Interval]] = None) -> 'Schedule':
+        """
+        Return a Schedule with only the instructions from this Schedule *failing* at least one of
+        the provided filters. That is, this function has the same inputs as self.filter, but returns
```

Comment:
```suggestion
        the provided filters. This method is the complement of `self.filter`, so that
            self.filter(args) | self.exclude(args) == self
```